scripts/completion/dvc.bash

```diff
@@ -10,33 +10,54 @@
 
 _dvc_commands='init destroy add import checkout run pull push fetch \
               status repro remove move unprotect gc config remote metrics \
-              install root lock unlock pipeline commit'
+              install root lock unlock pipeline commit cache cache_dir diff \
+              config index metrics_add metrics_modify metrics_remove move \
```

Comment:
I don't think we have an `index` command :)

Reply:
I am still using 0.40.7. I should upgrade it.